helm-configs/glance/glance-helm-overrides.yaml

```diff
@@ -367,6 +370,11 @@ conf:
     auth_address = {{ tuple "ceph_object_store" "public" "api" . | include "helm-toolkit.endpoints.keystone_endpoint_uri_lookup" }}
     user = {{ .Values.endpoints.ceph_object_store.auth.glance.username }}:swift
     key = {{ .Values.endpoints.ceph_object_store.auth.glance.password }}
+    {{- else if eq .Values.storage "swift" }}
+    auth_version = 1
```

Comment:
do we need to set the auth version here?

Reply:
Yeah, we can set it to 3 here and override to version=1 for tempauth from region specific config file.